conf/application.conf

```diff
@@ -117,9 +121,17 @@ auditing {
   }
 }
 
+google-analytics {
+  token=N/A
+  host=auto
+}
+
 
 assets {
   version = "2.253.0"
   version = ${?ASSETS_FRONTEND_VERSION}
   url = "http://localhost:9032/assets/"
 }
+
+
+stride.roles = [ ]
```

Comment:
can we have something here so that we can check the roles when running the service locally..?

Reply:
it would be more secure if this remained empty so that we don't accidentally allow roles where we don't want them to be, but I agree it is annoying to have to put in a value locally. I'll put in a role here which is obviously a test role.

Reply:
yeah, what ever we put here it will be overridden by the entry in the app-configs